OmiseSwiftTests/Fixtures/api.omise.co/charges-fpx-post.json

```diff
@@ -0,0 +1,92 @@
+{
+    "object": "charge",
+    "id": "chrg_5mx7ibz77gqegp7yew1",
+    "location": "/charges/chrg_5mx7ibz77gqegp7yew1",
+    "amount": 2500,
+    "net": 18568,
+    "fee": 0,
+    "fee_vat": 0,
+    "interest": 0,
+    "interest_vat": 0,
+    "funding_amount": 18568,
+    "refunded_amount": 0,
+    "authorized": false,
+    "capturable": false,
+    "capture": true,
+    "disputable": false,
+    "livemode": false,
+    "refundable": false,
+    "reversed": false,
+    "reversible": false,
+    "voided": false,
+    "paid": false,
+    "expired": false,
+    "platform_fee": {
+        "fixed": null,
+        "amount": null,
+        "percentage": null
+    },
+    "currency": "MYR",
+    "funding_currency": "THB",
```

Comment:
Small point, but this kind of response won't necessarily be representative of FPX charges as they are not multi-currency.